app/views/users/new.html.erb

```diff
@@ -0,0 +1,75 @@
+<%# <form method="POST" action="users_path" accept-charset="UTF-8"> %>
+    <%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>
+    <%# <table>
+        <tr>
+            <td><label for="user_name">Username: </label></td>
+            <td><input type="text" name="user[username]" id="user_name"></td>
+        </tr>
+        <tr>
+            <td><label for="email">Email: </label></td>
+            <td><input type="text" name="user[email]" id="email"></td>
+        </tr>
+        <tr>
+            <td><label for="password">Password: </label></td>
+            <td><input type="password" name="user[password]" id="password"></td>
+        </tr>
+    </table> %>
+
+    <%# <input type="submit" value="Submit"> %>
+<%# </form> %>
+
+<%# <%= form_tag("users_path", method: "post") do %> 
+    <%# <%= label_tag(:username, "Username: ")%>
+    <%# <%= text_field_tag(:username)%> 
+    <%# <%= label_tag(:username, "email: ")%> 
+    <%# <%= text_field_tag(:email)%> 
+    <%# <%= label_tag(:username, "password: ")%> 
+    <%# <%= password_field_tag(:password)%> 
+    <%# <%= submit_tag("Submit")%> 
+<%# <% end %> 
+
+  <%= form_for(@user) do |f| %>    
+    <table>
+    <tr>
+        <td>
+            <%= f.label :username, 'Your username here' %> 
+        </td>
+        <td>
+            <%= f.text_field :username %>
+        </td>
+    </tr>
+    <tr>
+        <td>
+            <li style="list-style: none; color: red"><%= @user.errors.full_messages[0] %></li>
+        </td>
+    </tr>
+
+    <tr>
+        <td>
+            <%= f.label :email %>
+        </td>
+        <td>
+            <%= f.text_field :email, placeholder:" email@example.com" %>
+        </td>
```

Comment:
This field should not be a text_field. It should be an email_field. This is important because it makes sure that what is added in the field is only an email address.